### PR TITLE
Visual+ Staffwho

### PR DESCRIPTION
**Pull request by realforest2001**
# About the pull request
Adds aesthetic ranks to staff who, so staff with multiple roles can be displayed. They only inherit the rights of their first one however.

# Explain why it's good for the game
Clarity for staff with multiple roles.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
add: StaffWho can now show multiple roles per staff member. This is purely aesthetical.
/:cl:

### CODE REVIEW

File: code/modules/admin/admin_ranks.dm [hunk 1]
```diff
@@ -104,6 +104,12 @@ var/list/admin_ranks = list()								//list of all ranks with associated rights
 	if(List.len >= 2)
 		rank = ckeyEx(List[2])
 
+	var/list/sranks = list()
+	if(List.len >= 3)
+		List -= List[2]
+		List -= List[1]
+		sranks = List
```
Comment (fira) [inline code/modules/admin/admin_ranks.dm:111]: What you want is probably `List.Copy(3)`

File: code/modules/admin/holder2.dm [hunk 2]
```diff
@@ -5,6 +5,7 @@ GLOBAL_PROTECT(href_token)
 
 /datum/admins
 	var/rank			= "Temporary Admin"
+	var/list/sranks		= null
```
Comment (fira) [inline code/modules/admin/holder2.dm:8]: Would be great to have a name somewhat more descriptive, all of that stuff is already cryptic as hell

File: code/modules/admin/holder2.dm [hunk 3]
```diff
@@ -21,7 +22,7 @@ GLOBAL_PROTECT(href_token)
 	///Whether this admin is invisiminning
 	var/invisimined = FALSE
 
-/datum/admins/New(initial_rank = "Temporary Admin", initial_rights = 0, ckey)
+/datum/admins/New(initial_rank = "Temporary Admin", initial_rights = 0, ckey, new_sranks)
```
Comment (fira) [inline code/modules/admin/holder2.dm:25]:
```suggestion
/datum/admins/New(initial_rank = "Temporary Admin", initial_rights = 0, ckey, list/new_sranks)
```
typing good for clarity

**Review by fira — CHANGES_REQUESTED** [review]
Looks good to me overall, though it also doesn't check ranks exists, so it's basically "additional titles"

**Comment by realforest2001** [comment]
> Looks good to me overall, though it also doesn't check ranks exists, so it's basically "additional titles"

Essentially for now yeah. In the future I may look at making it check against things to allow integration, but until then I think it should be ok.

**Pull request merged**